crates/uv-git/src/git.rs

```diff
@@ -22,8 +22,20 @@ use uv_static::EnvVars;
 /// checkout is ready to go. See [`GitCheckout::reset`] for why we need this.
 const CHECKOUT_READY_LOCK: &str = ".ok";
 
+#[derive(Debug, thiserror::Error)]
+pub enum GitError {
+    #[error("Git executable not found. Ensure that Git is installed and available.")]
+    GitNotFound,
+    #[error("Unexpected error: {0}")]
+    Other(String),
```

Comment:
I think we should make this `#[error(transparent)]` and pass the source unmodified.